docs/content/en/api.md

```diff
@@ -106,24 +105,29 @@ The key name for the content-holding property
 
 The key name for possible meta templates
 
-### refreshOnceOnNavigation <Badge text="runtime" type="yellow" />
+### refreshOnceOnNavigation
+<badge color="orange">runtime</badge>
```

Comment:
This might be an issue with the deploy preview but I dont see the orange color in the netlify build? All badges are still green?